R/ecr_metaepoch.R

```diff
@@ -29,6 +30,7 @@ ecr_metaepoch <- function(config_ecr) {
     params$initial.solutions <- matrix_to_list(suggestions)
     params$representation <- "float"
     params$monitor <- FALSE
+    params$terminators <- list(stopOnIters(max.iter = iterations_count * population_size))
```

Comment:
Uuu, z tym warunkiem stopu to działa lepiej? Czemu w ogóle domnażasz to razy `population_size` w tym `max.iter`? Ten parametr to jest tak głupio nazwana liczba ewaluacji?

Reply:
tak mi się wydaje, jak testowałem to właśnie tak to działało xd